app/models/exchange_rate_currency_rate.rb

```diff
@@ -76,6 +92,14 @@ def months_for_year(year)
         .uniq
     end
 
+    def for_month(month, year = Time.zone.today.year)
```

Comment:
You could make this extendable by moving it to the dataset module section and doing the `.all` call elsewhere. No biggie, though, we might never need variations on this scope I guess

Reply:
Yeah i couldnt think any more variations, will have a think though

Reply:
Yeah, fair enough